src/nnet3/nnet-am-decodable-simple.h

```diff
@@ -213,6 +219,10 @@ class NnetDecodableBase {
   // online_ivector_period_ helps us interpret online_ivector_feats_; it's the
   // number of frames the rows of ivector_feats are separated by.
   int32 online_ivector_period_;
+  // In the multiple ivectors case, it is the interval between two consecutive
+  // frames that we want to get ivectors for, so that we can dump multiple
+  // ivectors for a chunk. In the single ivector case, its value is just 0.
```

Comment:
ivector_/online_ivectors_ exists in the original code, so I think they are not intended to determine single/multiple ivectors case. 

In the config generation script, I use if ivector_interval == 0 to determine whether to use ReplaceIndex or Round for ivector. Also in nnet3-get-egs, I use the same condition to determine single/multiple ivector case.